### PR TITLE
New flag and timestamp

### PR DESCRIPTION
**Pull request by JuezFenix**

### CODE REVIEW

**Comment by JuezFenix** [comment]
There you go (or at least i created a wiki on my fork with the new changes). I have added another small change, which can be activated or deactivated the timestamp for notifications.

**Comment by pyed** [comment]
I think the timestamp feature is kinda useless as Telegram messages are timestamped, and the way we monitor the completion and addition of torrents is live, I don't see the reason to timestamp the message before sending it, what do you think ?

File: README.md [hunk 1]
```diff
@@ -20,6 +20,6 @@ And you should be good to go!
 
 ## Wiki
 
-* [Getting started](https://github.com/pyed/rtelegram/wiki)
-* [Commands](https://github.com/pyed/rtelegram/wiki/Commands)
-* [How to get notifications](https://github.com/pyed/rtelegram/wiki/Notifications)
\ No newline at end of file
+* [Getting started](https://github.com/juezfenix/rtelegram/wiki)
+* [Commands](https://github.com/juezfenix/rtelegram/wiki/Commands)
+* [How to get notifications](https://github.com/juezfenix/rtelegram/wiki/Notifications)
```
[the remark this answers is not in the document]
Comment (JuezFenix) [reply README.md]: Changed

**Comment by JuezFenix** [comment]
> I think the timestamp feature is kinda useless as Telegram messages are timestamped, and the way we monitor the completion and addition of torrents is live, I don't see the reason to timestamp the message before sending it, what do you think ?

I think you are partly rigth, but I think that could be usefull to have a timestamp from server side (could be used for problem debug or something).

**Pull request closed without merging**